server/src/main/java/org/opensearch/cluster/coordination/CoordinationState.java

```diff
@@ -52,6 +52,7 @@
 import java.util.Set;
 
 import static org.opensearch.cluster.coordination.Coordinator.ZEN1_BWC_TERM;
+import static org.opensearch.node.remotestore.RemoteStoreNodeAttribute.isRemoteRoutingTableEnabled;
```

Comment:
can we rename this to isRemotePublicationEnabled itself (it is controlling publication for both metadata & routing table)

Reply:
Will rename to isRemotePublicationEnabled